models/9var/README.rst

```diff
@@ -0,0 +1,100 @@
+#################
+9-variable README
+#################
```

Comment:
Some of the low-order models have a `workshop_setup.csh` that runs a canned experiment. The 9var model has one. Do we want to document that here? something for the group to decide. Then, do we want to do it now - surely we can add it later.

Reply:
Fixed.